deps/wrapper.cc

```diff
@@ -313,6 +320,31 @@ Status report_error(std::string *application_error,
     return worker.PushError(jobid, "task", err_msg, timestamp);
 }
 
+// Serialize a JobConfig protobuf message into its serialized string equivalent by constructing it
+// via it's JSON representation of the message. Most likely there is a better way to construct
+// the protobuf message but this is works for now.
```

Comment:
yeah, probably a better way to do this but given that we're not setting this per-task, I think it's fine if it's a bit inefficient and worth the tradeoff in not getting too rabbit holed here